api/v1/views/places_reviews.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python3
+from api.v1.views import app_views
+from flask import Flask, jsonify, abort, request
+from models import storage
+from models.place import Place
+from models.review import Review
+
+
+@app_views.route('/places/<place_id>/reviews',
+                 methods=['GET'], strict_slashes=False)
+def get_places_reviews_id(place_id):
```

Comment:
saw in other scripts that this definition could use a better name. Something like "get_places_id_reviews"

Reply:
Ok, change fixed